bugbug/utils.py

```diff
@@ -18,6 +18,27 @@
 
 TASKCLUSTER_DEFAULT_URL = "https://taskcluster.net"
 
+def zstd_compress(path):
```

Comment:
You should just use copy_stream, copying the preexisting code, why are you complicating your life?

Reply:
Are you just talking about the compress_file code or decompress one too?? Because decompress_file is not exactly decompressing the file when i tried running it on a seperate python script to test it.

Reply:
I'm talking about both, the goal of this PR is just to move the functions in utils.py, you shouldn't need to change anything inside the functions themselves.

Reply:
@marco-c Alright my bad! One last thing since when i first made the commit you told me to check if the functions are running or not, i tried testing them on a file. Compression was successful but I'm not sure if the decompress function was working so let me know if this is the correct one for zstd_decompress:

def zstd_decompress(path):
    dctx = zstandard.ZstdDecompressor()
    with open(f"{path}.zst", "rb") as input_f:
        with open(path, "wb") as output_f:
            dctx.copy_stream(input_f, output_f)

Reply:
> I'm not sure if the decompress function was working

You can simply run your function to make sure it works.